### PR TITLE
Make resizing work cross-origin

### PR DESCRIPTION
**Pull request by nottrobin**
I realised resizing doesn't work cross-origin, see the testing page.

This fixes it and also reduces the number of requests down to 1 (PERFORMANCE!!!)

QA
===

```
npm i
npm run build
npm start
```

Open <http://127.0.0.1:8080/testing/> and see all the heights are right.

### CODE REVIEW

File: js/example.js [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
-  if (!link) { return; }
+function renderExample(exampleElement) {
+  var link = exampleElement.href;
+  var request = new XMLHttpRequest();
+
+  request.onreadystatechange = function() {
+      if (request.status === 200 && request.readyState === 4) {
+        var html = request.responseText;
+        renderIframe(exampleElement, html);
+        renderCodeBlock(exampleElement, html);
```
Comment (anthonydillon) [inline js/example.js:12]: Nice way of falling back both parts if the link 404s.

File: js/example.js [hunk 2]
```diff
@@ -1,43 +1,55 @@
...
+  var resizeInterval = setInterval(
+    function() {
+      if (iframe.contentDocument.readyState == 'complete') {
+        iframe.height = iframe.contentDocument.body.scrollHeight + "px";
+        clearInterval(resizeInterval);
+      }
+    },
+    100
+  );
+  setTimeout(function() {clearInterval(resizeInterval);}, 2000);
```
Comment (anthonydillon) [inline js/example.js:42]: Nice catch with the timeout.

**Review by anthonydillon — COMMENTED** [review]
Code looks good. Why did you remove the hide link functionality at the end?

**Comment by nottrobin** [comment]
@anthonydillon I removed the `hideExample` function because I felt that creating a separate function for a single line of code which is used exactly once was overkill. But I don't feel strongly, can put it back if you like.

**Review by anthonydillon — APPROVED** [review]
Oh I see its there. thought it was removed all together.

**Pull request merged**